internal/parsers/db/point.go

```diff
@@ -13,26 +14,45 @@ func PointFromCoordinates(crd *Coordinates) (locodedb.Point, error) {
 		return locodedb.Point{}, nil
 	}
 
+	var lat, lng float64
+	var err error
+
 	cLat := crd.Latitude()
-	cLatDeg := cLat.Degrees()
-	cLatMnt := cLat.Minutes()
+	latString := string(cLat.value)
+	if strings.Contains(latString, ".") {
+		lat, err = strconv.ParseFloat(latString[:len(latString)-1], 64)
+		if err != nil {
+			return locodedb.Point{}, fmt.Errorf("could not parse latitude: %w", err)
+		}
+	} else {
+		cLatDeg := cLat.Degrees()
+		cLatMnt := cLat.Minutes()
 
-	lat, err := toDecimal(cLatDeg[:], cLatMnt[:])
-	if err != nil {
-		return locodedb.Point{}, fmt.Errorf("could not parse latitude: %w", err)
+		lat, err = toDecimal(cLatDeg[:], cLatMnt[:])
+		if err != nil {
+			return locodedb.Point{}, fmt.Errorf("could not parse latitude: %w", err)
+		}
 	}
 
 	if !cLat.Hemisphere().North() {
 		lat = -lat
 	}
 
 	cLng := crd.Longitude()
-	cLngDeg := cLng.Degrees()
-	cLngMnt := cLng.Minutes()
+	lngString := string(cLng.value)
```

Comment:
a good candidate for a separate function (lat and lng are parsed similarly) to prevent self-repeating
also, a unit test can be added for such exceptions